src/main.rs

```diff
@@ -70,7 +70,7 @@ fn with_games(games: Games) -> impl Filter<Extract=(Games, ), Error=Infallible>
     warp::any().map(move || games.clone())
 }
 
-fn with_word_generator(word_generator: impl WordGenerator) -> impl Filter<Extract=(dyn WordGenerator, ), Error=Infallible> + Clone {
+fn with_word_generator(word_generator: impl WordGenerator) -> impl Filter<Extract=(impl WordGenerator, ), Error=Infallible> + Clone {
```

Comment:
I think this one was just an oversight. `dyn MyTraitName` does not have time known at compile time. Not sure if `Box<dyn WordGenerator>` (allocating it on the heap) would work here, but it might.